client/html/templates/checkout/standard/process-body-standard.php

```diff
@@ -172,6 +172,9 @@
 		<?php endforeach; ?>
 	</ul>
 
+    <?php  ?>
```

Comment:
You can remove this now

Reply:
Done

Reply:
Not yet ;-)
Can you please do the change in the core so we can merge that afterwards?

Reply:
Hah :-) 
fixed!